docs/components/connectors/out-of-the-box-connectors/amazon-dynamodb.md

```diff
@@ -294,3 +296,17 @@ To avoid performance issues, it is recommended to limit the number of retries.
 :::
 
 To learn more about implementing retry logic in your BPMN diagram, you can refer to the [Camunda BPMN examples](https://camunda.com/bpmn/examples/) page, which includes examples of BPMN diagrams with timer and loop configurations.
+
+## Appendix
+
+### AWS authentication types
+
+There are 2 options of authenticating the **Connector** with AWS.
+
+- Choose `Credentials` option in the `Authentication` dropdown, if you have a valid pair of access and secret key provided by
+  your AWS account administrator. This option is applicable for both SaaS and self-managed option.
+- Choose `Default Credentials Chain (Hybrid/Self-Managed only)` option in the `Authentication` dropdown, if your system
+  is configured implicit authentication mechanism, such as role-based authentication, credentials supplied via environment
+  variables, or files on target host. Please keep in mind, that this option is applicable only for self-managed or hybrid
+  distribution. This approach uses the [Default Credential Provider Chain](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html)
+  to resolve required credentials.
```

Comment:
@igpetrov It would reduce the amount of repetetive docs if we could refer to a general auth guide for all AWS connectors wdyt? cc @christinaausley 

Maybe we can create a general `aws.md` file that contains this kind of information.

Reply:
@sbuettner, thought about it too yesterday, but I am just not sure what to put there. There is not enough of content as of now.

Reply:
@igpetrov Maybe authentication is enough. Or just include the content to not repeat it.

Reply:
Regardless, I don't think it makes sense to have a separate appendix section for just one topic. I would rather just nest that information directly beneath the related concept.

Reply:
Including "- Choose **Credentials** in the **Authentication** dropdown if you have a valid pair of access and secret key provided by your AWS account administrator. This option is applicable for both SaaS and Self-Managed users." feels repetitive based on the content outlined above this. Could always just add an admonition for the second option, or simply list it below.